exir/tests/test_memory_planning.py

```diff
@@ -868,18 +868,19 @@ def forward(self, input, label):
 
         ep.dump_executorch_program(True)
 
-        # 147 just so happens to be the index of the user_grad output arg of
+        # 149 just so happens to be the index of the user_grad output arg of
         # convolution_backward.out. This is fairly fragile.
         # Check that the None output is not memory planned.
+        self.assertEqual(len(ep.executorch_program.execution_plan[0].values), 151)
         self.assertEqual(
             ep.executorch_program.execution_plan[0]
-            .values[147]
+            .values[149]
```

Comment:
Not sure how this will pass w/o https://github.com/pytorch/pytorch/pull/144765. But if it does, then the test may not care about it in the first place :)